com.note_slayer.app.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.note_slayer.app
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+separate-locales: false
+rename-desktop-file: noteslayer.desktop
+rename-icon: noteslayer
+modules:
+  - name: noteslayer
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://github.com/X-SLAYER/note_slayer_public/releases/download/V0.1/note_slayer_linux.deb
+        sha256: 5aee6f197601b82b65355dad919477789b466a27ae7c8bbd33d19a90d0d01270
+        dest-filename: note_slayer_linux.deb
+      - type: script
+        dest-filename: noteslayer.sh
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - exec zypak-wrapper /app/noteslayer/noteslayer "$@"
+      - type: file
+        url: https://raw.githubusercontent.com/X-SLAYER/note_slayer_public/main/com.note_slayer.appdata.xml
```

Comment:
you should not pull this from `main` but from the tag / commit used to produce the binaries.

Also this should be in the .deb too so you don't have to do this.

Reply:
```
    <project_license>GPL-3.0</project_license>
```

So where is the source code?

Reply:
Te initial indication of GPL-3.0 was a mistake. NoteSlayer is not open source; it's under a proprietary license. Iv updated our metadata to reflect this

Reply:
It already has a checksum mismatch 

```
expected "c0c574bc221cf617c987e492ebe988ebe383fd3022f396292469afa7eabad330", was "71039350d1a4163b6c16a30e898cc45c79bceb847043c60c005f7ff428c2fd09"
```

Reply:
Fix these:

```
com.note_slayer.appdata.xml: FAILED:
• tag-invalid           : <project_license> is not valid [Proprietary]SPDX ID 'Proprietary' unknown
• style-invalid         : Too many <p> tags for a good description [11/10]
Validation of files failed
```